e2e/eip_1898_test.go

```diff
@@ -0,0 +1,239 @@
+package e2e
+
+import (
+	"context"
+	"github.com/0xPolygon/polygon-sdk/e2e/framework"
+	"github.com/0xPolygon/polygon-sdk/helper/tests"
+	"github.com/0xPolygon/polygon-sdk/jsonrpc"
+	"github.com/0xPolygon/polygon-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/umbracle/go-web3"
+	"math/big"
+	"testing"
+	"time"
+)
+
+func Test_GetBalance(t *testing.T) {
```

Comment:
E2E test are meant to test things in integration (consensus + txpool + syncer) but I think the JsonRPC server can be tested in unit tests.

Reply:
@PtitLuca 

I agree with Ferran, there's no reason this should be an integration test, since you're just testing out different queries for endpoints.

Please make it a unit test with a testing table that covers these cases 🙏

Reply:
@PtitLuca fixed it in df1f861 👍